qa/restart-upgrade/build.gradle

```diff
@@ -0,0 +1,87 @@
+/*
+ *  Copyright OpenSearch Contributors
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import org.opensearch.gradle.testclusters.StandaloneRestIntegTestTask
+import org.apache.tools.ant.taskdefs.condition.Os
+import java.util.concurrent.Callable
+
+apply from : "$rootDir/qa/build.gradle"
```

Comment:
Do we need to call this? Or will it be automatically inherited?

Reply:
We need to call this